azure-pipelines/integrationtests.yml

```diff
@@ -1,56 +1,67 @@
 pool:
-    name: SqlToolsTestAgentPool
-    demands: VisualStudio_16.0
+    name: ads-build-1es-hosted-pool
+    demands: 
+    - ImageOverride -equals ADS-Linux_Image
 
 steps:
-  - task: DownloadSecureFile@1
-    displayName: 'Download secure file'
+
+  - task: DockerInstaller@0
+    displayName: Installing Docker
     inputs:
       secureFile: 'testsettings.json'
+      dockerVersion: 17.09.0-ce
+      releaseType: stable
 
-  - task: VisualStudioTestPlatformInstaller@1
-    displayName: 'Visual Studio Test Platform Installer'
-    inputs:
-      versionSelector: latestStable
+  - script: docker pull mcr.microsoft.com/mssql/server:2019-latest
+    displayName: Pull MSSQL Docker Image  
+
+  - script: 'docker run -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=$(sql-server-password)"

```

Comment:
We should be generating the password in the pipeline, see the example pipelines I linked you